playbooks/group_vars/all.yml

```diff
@@ -0,0 +1,49 @@
+---
+upstream_repos_clone_folder: "/opt"
+suse_osh_registry_location: "{{ lookup('env','OPENSTACK_HELM_IMAGE_REGISTRY_LOCATION') | default('docker.io', true) }}"
+suse_osh_image_version: "{{ lookup('env','OPENSTACK_HELM_IMAGE_VERSION') | default('pike', true) }}"
```

Comment:
This is, for me, very brittle.
A) It means the group vars hold version, which is less semantically correct than the deploy-osh role. If we want a common variable, can we use something in the manifest?
B) We should make it very intentional to override the default branch, so it needs to be in extra vars IMO. On top of that, a deployer won't store in a file the current deployment version if it's defined as an env var (it ends up by reducing reproducibility for the env var case)
C) This is a new feature and should be added in a separate commit.